### PR TITLE
feat: meson ternaryoperator and buildin vairable

### PR DESCRIPTION
**Pull request by Decodetalkers**
? is the operator of ternaryoperator , so add ? to operator

buildin variable

### CODE REVIEW

File: queries/meson/highlights.scm [hunk 1]
```diff
@@ -36,6 +36,7 @@
   "-="
   ">"
   ">="
+	"?"
 ] @operator
```
Comment (theHamsta) [inline queries/meson/highlights.scm:40]: There was a discussion in the past on whether ternary should be `@operator` or `@conditional`. The `@conditional` side won (see C/C#/Java). I also preferred `@operator` back then. But we should keep consistency.

 We now query fallbacks. So now a `@conditional.ternary` would be possible if applied to consistently across languages

Comment (Decodetalkers) [reply queries/meson/highlights.scm:39]: So when will this keyword be available?

Comment (theHamsta) [reply queries/meson/highlights.scm:39]: Strange, the check was intended to allow everything that starts with `@conditional.Xxxx` let me check what's wrong

**Comment by Decodetalkers** [comment]
@theHamsta Since this pr have not merged, so I add something else, for the variable buildin vaiable, but I think keyword is not right, can you review it? 
Now it is what it is  looks like 
image

**Comment by tristan957** [comment]
Meson keywords look correct.

**Pull request closed without merging**